libs/tup-components/src/allocations/AllocationTable.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { LoadingSpinner, InlineMessage } from '@tacc/core-components';
+import { useProjects } from '@tacc/tup-hooks';
+import { formatDate } from '../utils/timeFormat';
+import styles from './Allocation.module.css';
+
+export const AllocationsTable: React.FC = () => {
+  const { data, isLoading, error } = useProjects();
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (error) {
+    return (
+      <InlineMessage type="warn">
+        Unable to retrieve allocation information.
+      </InlineMessage>
+    );
+  }
+
+  return (
+    <div>
```

Comment:
**Extra Markup (Maybe):** If you do not need this `<div>`, please do not add it. Possible use cases would be [`o-table-wrap ...` class(es)](https://dev.tup.tacc.utexas.edu/static/ui/components/detail/o-table-wrap--default.html). To remove this `<div>` without React complaining you need a single wrapping element, use `<>` and `</>` ([React fragment short syntax](https://reactjs.org/docs/fragments.html#short-syntax)).